Formula/singular.rb

```diff
@@ -0,0 +1,43 @@
+class Singular < Formula
+  desc "Computer algebra system for polynomial computations."
```

Comment:
No period at the end here please.

@DomT4 did the audit for that break?

Reply:
@ilovezfs I thought the PR for that died before being merged? As far as I'm aware we don't audit `desc` for ending with a period. We audit `keg_only` for that though, and I agree it wouldn't be awful if consistency was enforced upon `desc` too.

Reply:
Yeah, unless I missed a follow-up PR, seems to have died before merge: https://github.com/Homebrew/brew/pull/2795.

Reply:
>I thought the PR for that died before being merged?

ug

Reply:
rubocop: where PRs come to die.

Reply:
To be honest that PR, despite its demise, looks almost shippable. Wonder if OP could be talked into reopening it.

Reply:
Possibly. It would need to be rewritten though due to https://github.com/Homebrew/brew/pull/3349 and the heredoc changes but that seems like something OP could grok.

Reply:
If there's still interest and OP doesn't want to pick it back up I can always `brew pull` it locally so OP's contribution isn't wiped & then make the various few final tweaks as my own unique commit before resubmitting it. I'll give OP a little bit to confirm if they're a yay/nay on picking it back up if they wish to though.

Reply:
Yeah I'm sad we let stale bot close that PR, but not too sad since OP didn't object when it happened.

>I'll give OP a little bit to confirm if they're a yay/nay on picking it back up

:+1: